src/router.ts

```diff
@@ -32,13 +32,18 @@ export default new Router({
           name: 'intake',
           component: DailyIntake,
         },
+        {
+          path: 'days/:day/:meal/add',
+          name: 'add-food',
+          component: AddFood,
+        },
         {
           path: '/',
           redirect: {
             name: 'intake',
             params: {
-              day: `${moment(new Date()).format('YYYY-MM-DD')}`
-            }
+              day: `${moment(new Date()).format('YYYY-MM-DD')}`,
```

Comment:
Same suggestion as in other file, move date format to constant in separate file.

Reply:
Done